src/controller/java/src/matter/controller/Messages.kt

```diff
@@ -154,7 +155,36 @@ sealed class SubscriptionState {
 class WriteRequest(
   val attributePath: AttributePath,
   val tlvPayload: ByteArray,
-)
+  val jsonString: String? = null,
+  val dataVersion: UInt? = null
+) {
+  @Suppress("UNUSED_PARAMETER")
+  fun getEndpointId(wildcardId: Long): Long {
+    return attributePath.endpointId.toLong()
+  }
+
+  @Suppress("UNUSED_PARAMETER")
+  fun getClusterId(wildcardId: Long): Long {
+    return attributePath.clusterId.toLong()
+  }
+
```

Comment:
Use UInt for wildcardId, attributeId and clusterId

Reply:
This method is only used in JNI Layer. I will modify to private.